### DISCUSSION TITLE
how to disable highlight of the word under cursor, and that for all other occurrences of the word?

### DISCUSSION THREAD

**Discussion by hopezh** [LazyVim/LazyVim#1888] [Q&A] [answered]
May I ask how to disable the highlight of the word under cursor, and that for all other occurrences of the word in the buffer? 

I  found [one reference](https://www.reddit.com/r/AstroNvim/comments/zerm87/how_to_disable_highlighting_of_the_word_under_the/) , but I'm unable to get it to work by putting the suggested code in `init.lua`. So, where should the following code be inserted? 

Thanks.

```
return {
  lsp = {
    on_attach = function(client, bufnr)
      if client.server_capabilities.documentHighlightProvider then
        vim.api.nvim_del_augroup_by_name(vim.fn.printf("lsp_document_highlight_%d", bufnr))
      end
    end,
  },
}
```

**Comment by dpetka2001** [comment, marked as answer]
I believe this is caused by [vim-illuminate](https://www.lazyvim.org/plugins/editor#vim-illuminate). Just try to disable it to see if it's what you want.

**Reply by hopezh** [reply]
Yes, you're right. Thank you for your prompt help.
